jest.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   testEnvironment: '../jest.env.js',
   clearMocks: true,
   runner: 'groups',
+  testTimeout: 10000,
```

Comment:
I get failing tests because the `beforeAll()` steps time out on many occasions. So you'd argue for setting the timeouts there? My only issue is that when we realise it's not enough for some application, we would have to update this in a lot of places. I could use a const for this though.

Reply:
In the `CI` or also on the unit tests? How many tests are failing?

Reply:
on my local machine, actually. 68 are failing, 19 / 25 test suites

Reply:
Wow this is severe. Then keep it as is!